src/GPU3D.cpp

```diff
@@ -1721,14 +1889,18 @@ void GPU3D::ExecuteCommand() noexcept
 
         /*printf("[GXS:%08X] 0x%02X,  0x%08X", GXStat, entry.Command, entry.Param);*/
 
+
+
         switch (entry.Command)
         {
         case 0x10: // matrix mode
+            NewWriteFD(entry.Command, &entry.Param);
```

Comment:
I dont do it for every command, i exclude nops, and box and vector tests.
Should probably move it to before anyway, and filter them out before writing to file. if just because i might reuse the system for a 3d debugger later on